src/App.test.js

```diff
@@ -0,0 +1,8 @@
+import { render, screen } from '@testing-library/react';
+import App from './App';
+
+test('renders learn react link', () => {
+  render(<App />);
+  const linkElement = screen.getByText(/learn react/i);
+  expect(linkElement).toBeInTheDocument();
+});
```

Comment:
- [ ] Kindly make sure your app has unit tests for several components of the app and some pure functions in Redux code.

- [ ] Kindly make sure your App has integration tests using React Testing Library(you can perform a mocking data fetching and Redux Store connection).

- [ ] Kindly  make sure the tests you implemented are passing

Reply:
This issue is well resolved! ✔️